aws/pr_review/outputs.tf

```
@@ -0,0 +1,4 @@
+output "ecr_repository_url_pr_review" {
+  description = "URL of the Review environment ECR"
+  value       = aws_ecr_repository.pr_review_repository[0].repository_url
+}
```

Do we need this output for something else?  If yes, you'll need to tweak the value like so:
```suggestion
output "ecr_repository_url_pr_review" {
  description = "URL of the Review environment ECR"
  value       = var.env == "staging" ? aws_ecr_repository.pr_review_repository[0].repository_url : ""
}
```